app/controllers/pages_controller.rb

```diff
@@ -56,10 +56,22 @@ def front_page_events
     present_featured_events(AirtablePlaceholderService.call('Event List'))
   end
 
+  def present_featured_events(events)
```

Comment:
I feel like we should move all of the more complex presentation logic out to the helper. What's your take? I realize that I had originally only moved that one method, which seems inconsistent.

If we're going to abandon it, though, we could then also remove the `include` and the file itself.